pyanaconda/display.py

```diff
@@ -327,7 +327,13 @@ def setup_display(anaconda, options):
             do_startup_x11_actions()
         except (OSError, RuntimeError) as e:
             log.warning("X startup failed: %s", e)
-            stdout_log.warning(X_FAILURE_MESSAGE)
+            stdout_log.warning("X startup failed, falling back to text mode. There are multiple "
+                               "ways to help this issue:")
```

Comment:
Same suggestion - change to "help resolve this issue".

Reply:
Thank you, used in subsequent PR.